packages/core/package.json

```diff
@@ -11,7 +11,7 @@
     "redis:debug": "docker rm --force redis-commerce-sdk-debug; docker run -dp 6379:6379 --name redis-commerce-sdk-debug redis:5-alpine && docker exec -it redis-commerce-sdk-debug redis-cli monitor",
     "test": "nyc mocha \"test/**/*.test.ts\" --reporter=xunit --reporter-options output=../../reports_integration/core.xml",
     "test:cache": "nyc --check-coverage=false mocha \"test_integration/**/*.ts\" --reporter=xunit --reporter-options output=../../reports/core-integration.xml --retries 2 --timeout 5000",
-    "test:integration": "npm run test:cache",
+    "test:integration": "npm run redis:shutdown || (npm run redis && npm run test:cache)",
```

Comment:
```suggestion
    "test:integration": "npm run redis:shutdown; npm run redis && npm run test:cache",
```

Reply:
Will take care of it in next PRs for etag related checkins